host/sgx/sgx_enclave_common_wrapper.c

```diff
@@ -168,6 +179,52 @@ void* oe_sgx_enclave_create(
         enclave_error);
 }
 
+void* oe_sgx_enclave_create_ex(
+    void* base_address,
+    size_t virtual_size,
+    size_t initial_commit,
+    uint32_t type,
+    const void* info,
+    size_t info_size,
+    const uint32_t ex_features,
+    const void* ex_features_p[32],
+    uint32_t* enclave_error)
+{
+    _load_sgx_enclave_common();
+    if (ex_features)
+    {
+        /* Check for enclave_create_ex() in the current PSW installed. */
+        if (OE_OK !=
```

Comment:
```suggestion
        if (!_enclave_create_ex)
```

_enclave_create_ex is already populated (to a valid value or null) via _load_sgx_enclave_common. We don't need to lookup again.